### DISCUSSION TITLE
多语种模型，如何才能大幅改善英文语句的准确率？偶尔会出现字母错误之类

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#13630] [General]
多语种模型，如何才能大幅改善英文语句的准确率？偶尔会出现字母错误之类

**Comment by GreatV** [comment]
自己微调一下，加入识别错误的数据

**Comment by nissansz** [comment]
是自己训练的模型。微调很难改善到理想效果，中文可以，英文不行









--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: Wang Xin  ***@***.***>
 

 
 发送时间: 2024-08-09 23:36:23
 

 
 收件人:PaddlePaddle/PaddleOCR ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [PaddlePaddle/PaddleOCR] 多语种模型，如何才能大幅改善英文语句的准确率？偶尔会出现字母错误之类 (Discussion #13630)
 





 自己微调一下，加入识别错误的数据

**Reply by GreatV** [reply]
大力出奇迹，增加数据量。

**Comment by nissansz** [comment]
要什么样的文本行？真实单词句子？随机字母拼接？









--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: Wang Xin  ***@***.***>
 

 
 发送时间: 2024-08-10 07:49:16
 

 
 收件人:PaddlePaddle/PaddleOCR ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [PaddlePaddle/PaddleOCR] 多语种模型，如何才能大幅改善英文语句的准确率？偶尔会出现字母错误之类 (Discussion #13630)
 





 大力出奇迹，增加数据量。
